internal/redis/redis.go

```diff
@@ -115,28 +115,38 @@ func (r *Redis) NotifyReceive(ctx context.Context) ([]byte, error) {
 
 // ApplausePublish saves an applause for the user at a given time as unix time
 // stamp.
-func (r *Redis) ApplausePublish(userID int, time int64) error {
+func (r *Redis) ApplausePublish(meetingID, userID int, time int64) error {
```

Comment:
Does `meetingID` have no or an unknown type?

Reply:
Same as above (meaning the `r`)

Reply:
No. The type is `int`. If you skip the type of an argument, it gets the type of the next argument. For example

```go
func max(a, b int) int
```
takes two variables of type int and returns one int.